packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -221,7 +221,9 @@
 </div>
 ```
 
-## Usage with [Item][item] Component
```

Comment:
I will look into this. It seems like I introduced a bug to the upstream repo.

Reply:
Well, you are right. But having links in the titles may be a little odd.